projects/plugins/jetpack/modules/google-fonts/load.php

```diff
@@ -6,7 +6,7 @@
  */
 
 add_action(
-	'plugins_loaded',
+	'after_setup_theme',
```

Comment:
We currently hook with priority `999` because:

```php
	// Ensure the action is loaded after the late_initialization.
	// See projects/plugins/jetpack/class.jetpack.php.
```

By switching to the later `'after_setup_theme'` hook, does that mean we can use the default hook priority?

Reply:
Oh yes, you're right! The priority of the `late_initialization` is 1 instead of 100 now.

Reply:
`'after_setup_theme'` does seem like a better action to be using though, given this module is only about rendering themes.

https://wp-kama.com/hooks/actions-order

Reply:
I tried to hook the `jetpack_loaded` action but it seems the global styles have been initialized before that